app/controllers/case_information_controller.rb

```diff
@@ -97,10 +110,121 @@ def nomis_offender_id_from_url
     params.require(:nomis_offender_id)
   end
 
+  def stage1_errors
+    if case_information_params[:last_known_location].nil?
```

Comment:
really unsure about this 'removing' of errors. Not even sure if its 'allowed' by rails - my understanding is that model errors are mutable so that you can add to them, unsure what effect removing errors would have. 
It's not clear to me why this code is here and what problems it addresses? Maybe its related to the validations rules not being as precise as we need?